assets/training/model_evaluation/components/model_prediction/spec.yaml

```diff
@@ -3,7 +3,7 @@ name: model_prediction
 display_name: Model Prediction
 description: Generate predictions on a given mlflow model for supported tasks.
 
-version: 0.0.28
+version: 0.0.29
```

Comment:
We would need to bump up the version in the pipeline component and bump up the version of the pipeline component.

Reply:
Don't have enough context here to understand. Is there another file that needs to be modified?

Reply:
Per Teams discussion, incremented version in pipeline_component/spec.yaml .